resources/sts/4.14/sts_instance_controlplane_permission_policy.json

```diff
@@ -23,27 +23,64 @@
                 "elasticloadbalancing:CreateLoadBalancer",
                 "elasticloadbalancing:CreateLoadBalancerPolicy",
                 "elasticloadbalancing:CreateLoadBalancerListeners",
-                "elasticloadbalancing:CreateTargetGroup",
                 "elasticloadbalancing:ConfigureHealthCheck",
                 "elasticloadbalancing:DeleteListener",
                 "elasticloadbalancing:DeleteLoadBalancer",
                 "elasticloadbalancing:DeleteLoadBalancerListeners",
-                "elasticloadbalancing:DeleteTargetGroup",
                 "elasticloadbalancing:DeregisterInstancesFromLoadBalancer",
                 "elasticloadbalancing:DeregisterTargets",
-                "elasticloadbalancing:Describe*",
                 "elasticloadbalancing:DetachLoadBalancerFromSubnets",
                 "elasticloadbalancing:ModifyListener",
                 "elasticloadbalancing:ModifyLoadBalancerAttributes",
-                "elasticloadbalancing:ModifyTargetGroup",
-                "elasticloadbalancing:ModifyTargetGroupAttributes",
                 "elasticloadbalancing:RegisterInstancesWithLoadBalancer",
                 "elasticloadbalancing:RegisterTargets",
                 "elasticloadbalancing:SetLoadBalancerPoliciesForBackendServer",
                 "elasticloadbalancing:SetLoadBalancerPoliciesOfListener",
                 "kms:DescribeKey"
             ],
             "Resource": "*"
+        },
+        {
+            "Sid": "ReadPermissions",
+            "Effect": "Allow",
+            "Action": [
+              "elasticloadbalancing:Describe*"
```

Comment:
Does the controlplane use all describe actions? Can you list out the ones that are related to target groups?